spec/features/menage_users_by_admin.rb

```diff
@@ -33,5 +33,12 @@
     sleep 1
     expect(page).to have_content("example22@o2.pl")
   end
+
+  scenario 'admin can go to admin panel'  do
```

Comment:
It will be nice to also add feature spec which will test that regular users cannot see a link to the admin page.